modules/API/external/external_api.js

```diff
@@ -366,6 +366,29 @@ export default class JitsiMeetExternalAPI extends EventEmitter {
         return iframe.contentWindow.document.getElementById('largeVideo');
     }
 
+    /**
+     * Getter for participant specific video element in Jitsi Meet.
+     *
+     * @param {string} participantId - Id of participant to return the video for.
+     *
+     * @returns {HTMLElement|undefined} - The requested video.
+     */
+    _getParticipantVideo(participantId) {
+        const iframe = this.getIFrame();
+
+        if (!iframe
+                || !iframe.contentWindow
+                || !iframe.contentWindow.document) {
+            return;
+        }
+
+        if (participantId === this._myUserID) {
```

Comment:
Could you please add `typeof participantId === 'undefined'` use case? The reason is that maybe it will be useful to be able to retrieve the local video without keeping track of the id. Also the local id is initialized after the 'conference-joined' event.

Reply:
Done.